docs/user-docs/latest/content-types-builder/introduction-to-content-types-builder.md

```diff
@@ -1,11 +1,18 @@
 # Introduction to the Content-Types Builder
 
-::: warning 🚧 This section of the user guide is a work in progress. Stay tuned!
-<br>
-:::
-
 The Content-Types Builder is a core plugin of Strapi. It is a feature that is always activated by default and cannot be deleted. It is however only accessible when the application is in a development environment.
 
-Administrators can access the Content-Types Builder from _Plugins > Content-Types Builder_ in the main navigation of the admin panel. From there, it is possible to create and manage content-types, including collection types, single types and components.
+Administrators can access the Content-Types Builder from _Plugins > Content-Types Builder_ in the main navigation of the admin panel.
 
 ![Content-Types Builder interface](../assets/content-types-builder/content-types-builder.png)
+
+From the Content-Types Builder, administrators can create and manage content-types: collection types and single types but also components. All 3 are displayed as categories in the left side of the Content-Types Builder interface. In each category are listed all content-types and components that have already been created, configured and saved.
+
+From each category of the Content-Types Builder, it is possible to:
+
+- click on an existing content-type or component to access it and edit it (see [Editing and deleting content-types](saving-editing-deleting-content-types)),
```

Comment:
```suggestion
- click on an existing content-type or component to access it and edit it (see [Managing content-types](managing-content-types)),
```

You probably changed the section title at some point and forgot to update the link ;-)